code/components/gta-net-five/src/CloneExperiments.cpp

```diff
@@ -5040,6 +5040,40 @@ static HookFunction hookFunctionDiag([]
 	{
 		PedPoolDiagError();
 	});
-#endif
+#endif

```

Comment:
This part again makes me slightly concerned about Git/editor configuration (autocrlf is true, right?)